app/assets/stylesheets/header.scss

```diff
@@ -119,6 +119,19 @@
     }
   }
 
+  .dropdown-toggle {
+    align-items: center;
+    display: flex;
+
+    .user-name {
+      display: inline-block;
```

Comment:
I think this isn't needed, at least my browser (firefox and chrome) is telling me, that it's using `display: block` anyway and just ignores it.